homeassistant/components/remote/strings.json

```diff
@@ -28,7 +28,7 @@
   "services": {
     "turn_on": {
       "name": "[%key:common::action::turn_on%]",
-      "description": "Sends the power on command.",
+      "description": "Sends the turn on command to a device.",
```

Comment:
These are controlling entities, not devices.
But they could target a lot of other things, like areas.

We should remove "to a device" from this and other locations in this PR.

Reply:
Correct, these actions can target both devices and entities, plus several of them via areas and labels.

Again I took this from the online docs which say: "Under target, select the target device."

Perhaps we should first decide how to reword the toggle action below.

Reply:
OK, no problem. Then we make all three action descriptions consistent by removing "a device" from the description of the toggle command.

And the remaining fix is to make turn on and turn off consistent with both using "turn on / off" now.